File: app/src/main/java/com/example/rentalwheels/Moreinfo.java
package com.example.rentalwheels;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;

public class Moreinfo extends AppCompatActivity {

    ImageView backbtn;
    Button proceedbtn;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        overridePendingTransition(R.anim.abc_fade_in,R.anim.abc_fade_out);
        setContentView(R.layout.activity_moreinfo);

        backbtn = findViewById(R.id.backbtn);
        backbtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(Moreinfo.this, MapsActivity.class);
                startActivity(intent);
            }
        });

        proceedbtn = (Button) findViewById(R.id.proceedbtn);
        proceedbtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                Intent intent = new Intent(Moreinfo.this, PlaceorderActivity.class);
                startActivity(intent);
            }
        });



        getWindow().getDecorView().setSystemUiVisibility(View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR);

    }
}

File: app/src/main/java/com/msdc/rentalwheels/modules/Moreinfo.kt
package com.msdc.rentalwheels.modules

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import com.msdc.rentalwheels.HomeActivity
import com.msdc.rentalwheels.R

class Moreinfo : AppCompatActivity() {

    private lateinit var backbtn: ImageView
    private lateinit var proceedbtn: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        overridePendingTransition(R.anim.abc_fade_in, R.anim.abc_fade_out)
        setContentView(R.layout.activity_moreinfo)

        backbtn = findViewById(R.id.backbtn)
        backbtn.setOnClickListener {
            val intent = Intent(this@Moreinfo, HomeActivity::class.java)
            startActivity(intent)
        }

        proceedbtn = findViewById(R.id.proceedbtn)
        proceedbtn.setOnClickListener {
            val intent = Intent(this@Moreinfo, PlaceorderActivity::class.java)
            startActivity(intent)
        }

        window.decorView.systemUiVisibility = View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
    }
}